dimagi/superset
# Adds ivorycoast country map

### SUMMARY

Adds the Ivory Coast Country map.

The instructions were followed from the guide [here](https://superset.apache.org/docs/configuration/country-map-tools/#adding-a-new-country)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


Screenshots showing that the country is now avaialble in the country map chart.
![image](https://github.com/user-attachments/assets/8a02ef87-55b1-4ff3-b861-1cec46a388ca)

![image](https://github.com/user-attachments/assets/119bea73-d252-4134-8dd3-3badbed7c43e)


### TESTING INSTRUCTIONS

Create a new country map chart after rebuilidng the frontend and you should see the new country displayed.

### ADDITIONAL INFORMATION
